library/src/main/java/com/anjlab/android/iab/v3/SkuDetails.java

```diff
@@ -83,72 +77,65 @@ public SkuDetails(JSONObject source) throws JSONException
         priceLong = source.optLong(Constants.RESPONSE_PRICE_MICROS);
         priceValue = priceLong / 1000000d;
         priceText = source.optString(Constants.RESPONSE_PRICE);
-        subscriptionPeriod = parsePeriod(source.optString(Constants.RESPONSE_SUBSCRIPTION_PERIOD, "P"));
-        subscriptionFreeTrialPeriod = parsePeriod(source.optString(Constants.RESPONSE_FREE_TRIAL_PERIOD, "P"));
-        haveTrialPeriod = subscriptionFreeTrialPeriod.toStandardSeconds().getSeconds() != 0;
```

Comment:
I haven't renamed these files, this diff it's due to the line changes, in fact I have removed the `haveTrialPeriod` inserted in the previous PR https://github.com/anjlab/android-inapp-billing-v3/pull/281/ because it's pointless if we don't use joda, and here it's where it was crashing, because of the `toStandardSeconds()` that's not working for all formats.